.github/workflows/build-test.yml

```diff
@@ -81,7 +81,7 @@ jobs:
     strategy:
       matrix:
         jdk: [ 11, 8 ]
-        scala: [ 2.12.15, 2.13.8, 3.0.2 ] # Should be sync with Mergify conditions (.mergify.yml)
+        scala: [ 2.12.15, 2.13.7, 3.1.2-RC1-bin-20211222-c94b333-NIGHTLY ]
```

Comment:
Cedric, can you change only Scala 3 version without removing the comment and changing the `2.13.X` version?

Reply:
Could thanks the long time to revert, and so the conflicts introduced meanwhile ...